TSC-Charter.md

```diff
@@ -50,43 +67,54 @@ to continue attending TSC meetings as an observer.
 
 ## Section 5. Responsibilities of the TSC.  
 
-The TSC is responsible for all technical development within the Planetary Software Organization,
-including:
-
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Maintaining a list of affiliate projects
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and coding standards.
+The TSC exercises autonomy in setting up and maintaining procedures and policies,
+as well as management and administrative structures as it deems appropriate for the
+maintenance and operation of these projects and resources.
+
+The TSC is responsible for all technical development within the
+Planetary Software Organization, including:
+
+* Technical direction
+* Setting and maintaining governance rules for the conduct and make-up of the
+  TSC, Working Groups and other bodies within the TSC's domain
+* Creating new repositories and projects under the 
+  [_planetarysoftware_ GitHub organization](https://github.com/planetarysoftware)
+  as required
+* Maintaining a list of projects and working groups
```

Comment:
Isn't "maintaining a list of projects" too generally worded? Do we mean "Top-Level Projects"? Or is this meant to indicate a wider scope than TLPs?

Reply:
I don't think it is too general.  Certainly, our TLPs we need to keep track of, but at some point we may have a non-top-level project (not sure what it would be, but its possible).  In that case, seems like it would be our responsibility to keep track of it.

Of course, I think this 'list of projects' thing is an echo of the document this was sourced from, and maybe the right solution is to just delete it.  Maybe it is just expected that we'd keep a list of our projects, and we don't need to be explicit about it here?

Reply:
i think we could delete that bullet as it's kind of expected?

Reply:
I tend to agree.  I'll fix.

Reply:
i am uncertain about the best procedure on who and when to mark these conversations as "resolved". Is it when I agree with your response, I resolve it because it's fine, or is the idea that your response still includes an action, so you resolve it when that's done. Or can you even not resolve it, b/c i'm in the role of reviewer now? Didn't use this process much yet...

Reply:
I'm also not certain.  I think that since you raised a conversation, there's something that you need to be satisfied.  So I think I would tend to think that once you're happy with my response (either via what I push to the PR or via an argument I make), then you should be the one to 'Resolve conversation' as that indicates that your concern has been addressed.